sklearn/tests/test_common.py

```diff
@@ -144,6 +144,12 @@ def test_configure():
         os.chdir(setup_path)
         old_argv = sys.argv
         sys.argv = ['setup.py', 'config']
+
+        # disable OpenMP checks for the tests since sklearn can be built and
+        # tested on different machines.
```

Comment:
I am a bit confused this comment. It reads as "disable openMP for all tests", but that's not the case is it? It's just for this `test_configure` test.

Also I don't exactly understand  why this is needed

Reply:
the openmp support check does a compilation. This should not be involved in configure. In configure, the cythonization is done but not the compilation. The thing is I wanted to do the openmp check before the cythonization because this step takes some time and I think you want to know as fast as possible if the build fails.